manim/utils/file_ops.py

```diff
@@ -77,3 +78,22 @@ def open_file(file_path, in_browser=False):
             raise OSError("Unable to identify your operating system...")
         commands.append(file_path)
         sp.Popen(commands)
+
+
+def open_media_file(file_writer):
```

Comment:
Question (if minor): Why not passing a parameter in open_media_file to either preview or show the file in the browser instead of using the config ?

That's a suggestion, that's fine to me like that but I just wanted to suggest it.

Reply:
Because whether or not the parameter got passed into an open media file, would rely on the same check anyway.

Reply:
I can understand